src/apps/reservation-list/modal/pause-reservation/pause-reservation.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { FC, useCallback, useState, useEffect } from "react";
+import { Link } from "../../../../components/atoms/link";
+import Modal, { useModalButtonHandler } from "../../../../core/utils/modal";
+import { useText } from "../../../../core/utils/text";
+import DateInput from "./date-input";
+import { useUpdateV5 } from "../../../../core/fbs/fbs";
+import { PatronV5 } from "../../../../core/fbs/model";
+import { getModalIds } from "../../../../core/utils/helpers/general";
+import { useConfig } from "../../../../core/utils/config";
+
+interface PauseReservationProps {
+  id: string;
+  user: PatronV5;
+}
+
+const PauseReservation: FC<PauseReservationProps> = ({ id, user }) => {
+  const t = useText();
+  const { mutate } = useUpdateV5();
+  const { close } = useModalButtonHandler();
+  const { pauseReservation } = getModalIds();
+  const config = useConfig();
+  const [startDate, setStartDate] = useState<string>(
+    (config("pauseReservationStartDateConfig") as string) || ""
+  );
+  const [endDate, setEndDate] = useState<string | null>("");
+
+  const save = useCallback(() => {
+    if (user && startDate && endDate) {
+      mutate(
+        {
+          data: {
+            patron: {
+              preferredPickupBranch: user.preferredPickupBranch,
+              receiveEmail: user.receiveEmail,
+              receivePostalMail: user.receivePostalMail,
+              receiveSms: user.receiveSms,
+              onHold: { from: startDate, to: endDate }
+            }
+          }
+        },
+        {
+          onSuccess: () => {
+            close(pauseReservation as string);
+          },
+          // todo error handling, missing in figma
+          onError: () => {}
+        }
+      );
+    }
```

Comment:
We do this in the reservation modal as well. I think we should use the same function:
https://github.com/reload/dpl-react/blob/eeffaae91f848f662bcffa4563169b2f78cd014c/src/components/reservation/forms/ModalReservationFormText.tsx#L61-L87
So we could make the unSubmit function in the link I wrote and turn it into a global patron data submit handler.
One important thing that is happening is that the QueryClient is feeded with the updated user data. And that results in two things:
* We get updated user data next time we request it in the application (otherwise we would get old user data from the cache)
* Since the QueryClient gets the updated user data, the next time we request the user data, it will be served from the cache and an actual request is not being performed